tools/pd-ctl/pdctl/ctl_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pdctl
+
+import (
+	"testing"
+
+	"github.com/spf13/cobra"
+)
+
+func newACommand() *cobra.Command {
+	a := &cobra.Command{
+		Use:   "testa",
+		Short: "test A command",
+	}
+	return a
+}
+
+func newBCommand() *cobra.Command {
+	a := &cobra.Command{
+		Use:   "testb",
+		Short: "test b command",
+	}
+	return a
+}
+func newDEFCommand() *cobra.Command {
+	a := &cobra.Command{
+		Use:   "testdef",
+		Short: "test def command",
+	}
+	return a
+}
+
+func newCCommand() *cobra.Command {
+	a := &cobra.Command{
+		Use:   "testc",
+		Short: "test c command",
+	}
+	return a
+}
```

Comment:
I think these code can be simplified as below:

```golang
func newCommand(usage, short string) *cobra.Command {
	a := &cobra.Command{
		Use:   usage,
		Short: short,
	}
	return a
}
```

Reply:
got it